ui/src/tasks/actions/index.ts

```diff
@@ -19,13 +18,23 @@ import {
   taskCloneFailed,
   taskRunSuccess,
   taskGetFailed,
-} from 'src/shared/copy/notifications'
-import {
   importTaskFailed,
   importTaskSucceeded,
 } from 'src/shared/copy/notifications'
 import {createTaskFromTemplate as createTaskFromTemplateAJAX} from 'src/templates/api'
-
+import {addDefaults} from 'src/templates/utils/'
+import {
+  deleteTask as apiDeleteTask,
+  deleteTasksLabel,
+  getTask as apiGetTask,
+  getTasks as apiGetTasks,
+  getTasksRuns,
+  getTasksRunsLogs,
+  patchTask,
+  postTask,
+  postTasksLabel,
+  postTasksRun,
+} from 'src/client'
```

Comment:
I hear what you're saying and had originally set out to use the same manner of importing and using the client routes, however @drdelambre brought up the point of being explicit and I felt it to be a good convention to follow:

https://github.com/influxdata/influxdb/pull/16325#discussion_r361754964

At the end of the day, this is a matter of preference, and while having an identifier like `api` before the function call name is a bit more indicative of the functionality we're performing, I don't think that importing the entire file to use one function from that file should serve that purpose. I am happy to alias the functions to their `apiFunctionName` though since that would help provide greater clarity on what the purpose of the functions are

Reply:
ok apiFunctionName it is then, yeah- opinions may differ :)